src/Engine/SpellFxRenderer.h

```diff
@@ -58,14 +62,13 @@ struct SpellFX_Billboard {
 /*  122 */
 #pragma pack(push, 1)
 struct PlayerBuffAnim {
-    inline PlayerBuffAnim()
-        : bRender(false), uSpellAnimTime(0), uSpellIconID(0) {}
-
-    int16_t bRender = 0;
-    int16_t field_2 = 0;
-    int uSpellAnimTimeElapsed = 0;
-    int uSpellAnimTime = 0;
-    int uSpellIconID = 0;
+    inline PlayerBuffAnim() : bRender(false), field_2(0), uSpellAnimTimeElapsed(0), uSpellAnimTime(0), uSpellIconID(0) {}
```

Comment:
Since you're changing it -- just move all initialization into fields and drop the ctor.